fet.sh

```diff
@@ -100,7 +100,7 @@ read -r model < /sys/devices/virtual/dmi/id/product_name
 [ $# -gt 0 ] && pkgs=$#
 
 print() {
-	[ "$2" ] && printf '\033[3%sm%6s\033[0m ~ %s\n' "${accent:-4}" "$1" "$2"
+	[ "$2" ] && printf '\033[9%sm%6s\033[0m%b%s\n' "${accent:-4}" "$1" "${separator:-" ~ "}" "$2"
```

Comment:
> "${separator:-" ~ "}"

You don't need to quote inside the quote.

Reply:
it looks so weird without it tho :/ but okay i guess

Reply:
disagree